src/test/clojure/clara/performance/test_rule_execution.clj

```diff
@@ -0,0 +1,39 @@
+(ns clara.performance.test-rule-execution
```

Comment:
Why is this Clojure (JVM)-only?

Reply:
I put this in a JVM only location because the cljs test took 490 seconds, and it didn't seem that it would provide much value.

Reply:
I think that this is the only outstanding comment, @WilliamParker  do you feel strongly that this should also run for CLJS?